contracts/Vault.sol

```diff
@@ -415,7 +407,7 @@ contract Vault is
         _transferAndCheckInputToken(msg.sender, _inputToken, _amount);
         uint256 underlyingAmount = _swapIntoUnderlying(_inputToken, _amount);
 
-        deposits[tokenId] = Deposit(underlyingAmount, 0, lockedUntil, 0);
+        deposits[tokenId] = Deposit(underlyingAmount, address(0), lockedUntil, 0);
```

Comment:
⚠️ assuming this is ok save address zero as claimer for sponsored deposits since it seems to be used as placeholder anyways, correct?